From foo@baz Sun Nov 19 11:32:28 CET 2017
From: Sasha Levin <alexander.levin@verizon.com>
Date: Thu, 16 Nov 2017 21:04:58 -0500
Subject: Revert "dt-bindings: Add LEGO MINDSTORMS EV3 compatible specification"

From: Sasha Levin <alexander.levin@verizon.com>


This reverts commit 852bf68b7a62195c3c0c63f3b11f3f30958fc220.

As Ben pointed out, these drivers don't exist in <=4.9.

Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 Documentation/devicetree/bindings/arm/davinci.txt |    4 ----
 1 file changed, 4 deletions(-)

--- a/Documentation/devicetree/bindings/arm/davinci.txt
+++ b/Documentation/devicetree/bindings/arm/davinci.txt
@@ -13,10 +13,6 @@ EnBW AM1808 based CMC board
 Required root node properties:
     - compatible = "enbw,cmc", "ti,da850;
 
-LEGO MINDSTORMS EV3 (AM1808 based)
-Required root node properties:
-    - compatible = "lego,ev3", "ti,da850";
-
 Generic DaVinci Boards
 ----------------------
 
